src/ralph/accounts/admin.py

```diff
@@ -85,6 +86,12 @@ def url(self, item):
         )
     url.title = _('Link')
 
+    def buyout_date(self, item):
+        return Asset.get_buyout_date(
+            item['invoice_date'], item['depreciation_rate']
```

Comment:
Can anyone confirm that this is really `months` because tooltip says something different.